core/src/com/fallenflame/game/LevelSelectMode.java

```diff
@@ -1,8 +1,6 @@
 package com.fallenflame.game;
 
-import com.badlogic.gdx.Input;
-import com.badlogic.gdx.InputProcessor;
-import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.*;
```

Comment:
Why *? Why are we importing so much?

Reply:
That is a good question. Not sure how that happened, I blame Intellij lol.

Good catch